packages/react-native-codegen/src/parsers/__tests__/parsers-primitives-test.js

```diff
@@ -829,7 +830,7 @@ describe('emitUnion', () => {
         'ObjectTypeAnnotation',
       ];
       describe('when nullable is true', () => {
-        it('throws an excpetion', () => {
+        it('throws an exception', () => {
```

Comment:
Just caught a spelling error scanning through the file as I was making my additional test